src/modules/clear_completed.js

```diff
@@ -0,0 +1,13 @@
+const clearCompleted = (toDoList, manageTasks) => {
+  const toDoItems = toDoList.querySelectorAll('.to-do-item');
+  toDoItems.forEach((toDoItem) => {
+    const checkButton = toDoItem.querySelector('.check-button');
+    if (checkButton.checked) {
+      const index = toDoItem.dataset.index - 1;
+      manageTasks.removeTask(index);
+      toDoItem.remove();
+    }
```

Comment:
- [ ] Great job on deleting the task on the UI once the `clear all` is pressed, however Kindly make sure the deleted tasks are also deleted in the `localStorage` because if you may see very well if the page is refreshed all the tasks come back up again. please fix that.